models/modelsmodule.h

```diff
@@ -29,10 +29,17 @@
 // Includes from sli:
 #include "slimodule.h"
 
+#include "nest.h"
```

Comment:
Add  `// Includes from nestkernel:` on line above and move section before includes from SLI